tests/arch/arm/arm_thread_swap/src/arm_thread_arch.c

```diff
@@ -52,6 +68,7 @@ static void load_callee_saved_regs(const _callee_saved_t *regs)
 		: "r" (regs)
 		: "memory"
 	);
+#endif
```

Comment:
Add
```
#else
#error Unsupported platform
```
? Or is that inappropriate/handled elsewhere?  (I don't see a need to do this in more than one of the places where exactly this conditional appears.)

Reply:
Right. I'll, perhaps do that in the first occasion in the file.
The test is anyway filtered-in only for Cortex-M architecture.

Reply:
Addressed @pabigot